lib/index.js

```diff
@@ -296,9 +309,26 @@ exports.parseGeneral = BBPromise.method(function(chtml){
 	// Copy key-value pairs with defined values to meta
 	var meta = {};
 	var value;
+	var notEmpty = false;
 	Object.keys(clutteredMeta).forEach(function(key){
+
 		value = clutteredMeta[key];
-		if (value){ // Only add if has value
+		var innerValue;
+		if (value !== null && typeof value === 'object'){
+			var arrlength = value.length;
```

Comment:
Objects don't have the property length. You'd need to do Object.keys(value).length